### DISCUSSION TITLE
DolphinScheduler of docker-compose install error

### DISCUSSION THREAD

**Discussion by rwrwd7** [apache/dolphinscheduler#12547] [Q&A]
<img width="709" alt="image" src="https://user-images.githubusercontent.com/7921289/197999475-96850764-631e-4a36-a97c-360ec72db61b.png">

RT.
As you can see from the picture, the word "存储未启用" is prompted. Please tell us how to solve it。

**Comment by DarkAssassinator** [comment]
hi @rwrwd7, u should exec docker container, and modify `resource.storage.type` in `common.properties`.

**Reply by rwrwd7** [reply]
Thanks, I get this properties.
